src/main/java/iabudiab/maven/plugins/dependencytrack/suppressions/Suppression.java

```diff
@@ -20,7 +20,7 @@
 public interface Suppression {
 
 	@JsonProperty("by")
-	String getType();
+	String type = null;
```

Comment:
Otherwise jackson duplicates the json property "by"

Reply:
Reverted change and removed lombok from interface implementations

Reply:
Removing lombok @Data annotation was not necessary. Reverted change. Fix was to specify `include = JsonTypeInfo.As.EXISTING_PROPERTY`  in `@JsonTypeInfo()` on Supression interface.